ckanext/switzerland/helpers/mail_helper.py

```diff
@@ -5,14 +5,25 @@
 from ckan.lib.base import render_jinja2
 import ckan.lib.mailer as mailer
 
+SITE_TITLE = config.get('ckan.site_title', 'opendata.swiss')
+SITE_URL = config.get('ckan.site_url', 'https://opendata.swiss')
+SHOWCASE_ADMIN_EMAIL = config.get(
+    'ckanext.switzerland.showcase_admin_email',
+    'opendata@bfs.admin.ch'
+)
+SHOWCASE_ADMIN_NAME = config.get(
+    'ckanext.switzerland.showcase_admin_name',
```

Comment:
Could you add documentation about the new config fields in the README, please? (I just checked, and we do have a few others that are not documented there, but it would still be nice not to add to that number. :sweat_smile: )